API-3875

cjrowe opened the pull request.

lburgos commented on app/uk/gov/hmrc/thirdpartyapplication/services/ApplicationService.scala:
```diff
@@ -137,7 +137,7 @@ class ApplicationService @Inject()(applicationRepository: ApplicationRepository,
 
       def updateWso2Subscriptions(): Future[Seq[HasSucceeded]] = {
-        apiGatewayStore.getSubscriptions(app.wso2Username, app.wso2Password, app.wso2ApplicationName) flatMap { originalApis =>
+        subscriptionRepository.getSubscriptions(applicationId) flatMap { originalApis =>
```
I don't think we need this change, as this entire method (updateWso2Subscriptions) will be removed once we switch over to AWS (updating an app to another tier won't require to update the subscriptions).

cjrowe replied:
Have reverted that now

lburgos commented:
Somewhat related to this PR, it may be a good idea to delete these 2 that are not needed since we changed the way we deal with subscriptions in AWS:

https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L146

and 

https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L161

cjrowe commented:
> Somewhat related to this PR, it may be a good idea to delete these 2 that are not needed since we changed the way we deal with subscriptions in AWS:
> 
> https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L146
> 
> and
> 
> https://github.com/hmrc/third-party-application/blob/master/app/uk/gov/hmrc/thirdpartyapplication/services/ApiGatewayStore.scala#L161

Removed those references now...
